code/controllers/subsystem/processing/actionstate.dm

```diff
@@ -0,0 +1,5 @@
+//Handles the state_process() side of action_states
+PROCESSING_SUBSYSTEM_DEF(actionstate)
+	name = "AI Action States Processing"
+	wait = 1
```

Comment:
```suggestion
	wait = 2
```

Reply:
Recall discussion in discord about the wait being the way it is for this review